app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions.unauthorisedPayments
+
+import models.pension.charges.UnauthorisedPaymentsViewModel
+import play.api.i18n.Messages
+import play.api.mvc.Call
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import utils.ViewUtils
+
+object UnauthorisedPaymentsCYAViewHelper {
```

Comment:
I would maybe have this in a helper directory 
From: [app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala]
To This: app/controllers/pensions/unauthorisedPayments/**helper**/UnauthorisedPaymentsCYAViewHelper.scala]

Reply:
OK.

Reply:
Though I think `helpers` would be more consistent.

Reply:
I've changed my mind.

As discussed, let's keep it as is, and revisit when we add another similar helper.